Tests/Blazorise.Tests/Components/ValidateAnnotationsComponentTest.cs

```diff
@@ -209,4 +210,27 @@ public void CanManuallyValidateName_InitiallyPopulated()
         btn.Click();
         Assert.Contains( "is-valid", edit.ClassList );
     }
+
+    [Fact]
+    public void CanAutoValidateName_InitiallyPopulated_ErrorOverride()
```

Comment:
I think the test is fine. 

I'd probably just add an additional one with an actual input other then **string.empty** for the invalid check?

Only thing of note is that if there is any race condition we need the usual `WaitForAssertion` helpers from bUnit. 
(Usually because the pipeline is more susceptible to it, but I think it's fine here.)

Reply:
Some tests are failing randomly, and I'm not sure if the problem is in the priority change or they didn't work before. Can you confirm?

Reply:
Heyia,

There's two tests failing because it can't locate the validationfeedback by text:
![image](https://github.com/Megabit/Blazorise/assets/22283161/72a7f7f3-b57d-496d-a633-92506bc9d2e9)

Is there a reason why you had removed it?

I reintroduced it.